pallets/common/src/traits/relayer.rs

```diff
@@ -1,14 +1,29 @@
 use crate::{traits::identity, CommonConfig};
-use frame_support::{decl_event, weights::Weight};
 use polymesh_primitives::{Balance, EventDid};
 
```

Comment:
General point: The way the code-base has been moving towards is just smashing together imports and letting rustfmt deal with it.